upgrading.adoc

```diff
@@ -416,44 +417,44 @@ known deployment mechanism.
 [[upgrading-risks]]
 === Risks
 
-During activation of any fork, hard or soft, miner activated or user
-activated, there's a risk of a long-lasting chain split. A split that
+During the activation of any fork, be it hard or soft, miner activated or user
+activated, there's the risk of a long-lasting chain split. A split that
 lingers for more than a few blocks can cause severe damage to the
-sentiment around Bitcoin and also to it's price. But above all, it
+sentiment around Bitcoin as well as to its price. But above all, it
 would cause great confusion over what Bitcoin is. Is Bitcoin this
 chain or that chain?
 
-The risk with a user activated soft fork is that the new rules
-activate even if the majority of the hash power doesn't support
-them. This scenario would result in a long lasting chain split that
-would remain until the majority of the hash power adopts the new
+The risk with a user activated soft fork is that the new rules get
+activated even if the majority of the hash power doesn't support
+them. This scenario would result in a long-lasting chain split, which
+would persist until the majority of the hash power adopts the new
 rules. It could be especially hard to incentivize miners to switch to
 the new chain if they had already mined blocks after the split on the
 old chain, because by switching branch they would be abandoning their
-own block rewards. However, contrary to this incentive, in March 2013,
-when a <<march2013split,long-lasting split>> occurred due to an
-unintentional hard fork, two major mining pools made the decision to
-abandon their branch of the split to restore consensus.
+own block rewards. However, it's worth mentioning a remarkable episode: in March 2013
+a <<march2013split,long-lasting split>> occurred due to an
```

Comment:
Should we keep the link text? Otherwise we can rewrite as

`a long-lasting split, explained in <<march2013split>>, occurred due to an`